### DISCUSSION TITLE
.traineddata custom file path

### DISCUSSION THREAD

**Discussion by Hyporos** [naptha/tesseract.js#960] [Q&A] [answered]
How can I change the file path for the .traineddata?

Thanks!

**Comment by Balearica** [comment]
The `langPath` argument specifies where `.traineddata` files are fetched from.  This is documented [here](https://github.com/naptha/tesseract.js/blob/master/docs/api.md#createworkeroptions-worker) and [here](https://github.com/naptha/tesseract.js/blob/master/docs/local-installation.md#langpath).

**Reply by Hyporos** [reply]
Ah sorry, I meant how I can change the path that the .traineddata is saved to? Right now its saved to the /src of my project and I can't figure out a way to change that. Is it with dataPath?

**Reply by Hyporos** [reply, marked as answer]
Nevermind, figured it out using cachePath
